CONTRIBUTING.md

```diff
@@ -90,12 +87,34 @@ to `types-foo-x.y.n+1`.
 *Note:* In its current implementation, typeshed cannot contain stubs for
 multiple versions of the same third-party library.  Prefer to generate
 stubs for the latest version released on PyPI at the time of your
-stubbing. The oldest version of the library for which the stubs are still
-applicable (i.e. reflect the actual runtime behaviour) can be indicated
-in `METADATA.toml` as `version = "x.y"`. Note that only two most significant
-version levels are supported (i.e. only single dot). When a significant change
-is made in the library, the version of the stub should be bumped (note that
-previous versions are still available on PyPI).
+stubbing.
+
+#### The `METADATA.toml` file
+
+The metadata file describes the stubs package using the
+[TOML file format](https://toml.io/en/). Currently, the following keys are
+supported:
+
+* `version`: The oldest version of the library for which the stubs are still
+  applicable (i.e. reflect the actual runtime behaviour). Note that only two
+  most significant version levels are supported (i.e. only single dot).
+  When a significant change is made in the library, the version of the
```

Comment:
This sentence is a bit confusing to me. If the "significant change" is adding new functionality, it's probably not that important to bump the typeshed version. I think more guidance on how to decide to bump the version would be useful.

Reply:
Maybe we should simplify this paragraph: Just something like:

> The version of the upstream library for which these stubs were written. Note that the version number can contain only one dot, e.g. "2.1" or "2021.5", but not "2.1.4" or "2021.5.13".

Reply:
Reading this over again I think the current wording actually makes sense: you write stubs for the latest version, and set the minimum version to the earliest version that is sufficiently compatible with the current version. We can revisit the wording if it becomes a source of confusion for contributors.